### ISSUE TITLE
Input data

### ISSUE THREAD

**Issue by leesunfreshing** [rubenvillegas/iclr2017mcnet#4]
Hi, according to the paper, the input are the feature maps till the 3rd pooling layer from VGG-16. I understand the motion_enc serves this purpose. I am wondering the pre-trained model includes the weights from pretrained VGG-16 on imagenet or you train it from scratch? Thanks for your time

**Comment by rubenvillegas** [comment]
Hi @leesunfreshing ,

Thanks for your question. All the models are trained from scratch. In the paper we mean that we use the VGG-16 architecture, but not the pre-trained weights. Please let me know if you have more questions.

Regards,
Ruben

**Comment by leesunfreshing** [comment]
Thx 4 your reply! I do have one more question. Does The L_GAN and L_img are both computed frame by frame, and then sum up? Just like the Deep multi scale paper, which uses the same loss function? Thx again.

**Comment by rubenvillegas** [comment]
L_img is computed frame by frame and then summed up. L_GAN is computed on the entire sequence.
